bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccHandler.java

```diff
@@ -183,6 +220,7 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                             } else {
                                 logger.debug("Command has wrong type, QuantityType or DecimalType required!");
                             }
+                            updateChannelsVehicle(vehicleName);
```

Comment:
Probably better to call the update methods at the end of the if statement of the vehicle, loadpoints and general channel groups.

Reply:
removed, as the function calls `refresh();` at the end, which is updating all channels.